Mail/mimePart.php

```diff
@@ -849,11 +849,16 @@ public static function encodeHeader($name, $value, $charset = 'ISO-8859-1',
             $charset = 'ISO-8859-1';
         }
 
+        $mb = function_exists('mb_convert_encoding');
```

Comment:
Maybe check if $charset != 'UTF-8' here, so we don't call mb_convert_encoding() if not needed.